modules/ROOT/pages/clauses/merge.adoc

```diff
@@ -86,6 +86,45 @@ A new node is created because there are no nodes labeled `Critic` in the databas
 | (:Critic) | ["Critic"]
 |===
 
+[[merge-merge-single-node-with-multiple-labels]]
+=== Merge single node with multiple labels
+
+Multiple labels are separated by colons:
+
+.Query
+[source, cypher]
+----
+MERGE (robert:Critic:Person)
+RETURN robert, labels(robert)
+----
+
+A new node is created because there are no nodes labeled both `Critic` and `Person` in the database:
+
+.Result
+[role="queryresult",options="header",cols="2*<m"]
+|===
+| robert | labels(robert)
+| (:Critic:Person) | ["Critic","Person"]
```

Comment:
I understand nodes aren't actually represented like this when returned, but the way it shows here makes using labels(robert) look redundant, I would return only the labels to nail the point